source/_integrations/sensor.rest.markdomain

```diff
@@ -0,0 +1,513 @@
+---
```

Comment:
We are trying to get away from platform-specific pages. If anything, all other platforms should be merged into `rest.markdown`.

Reply:
That makes sense. If they get to one of the pages they might not discover both options.  Do you want me to merge them in this PR or in a followup?

Reply:
I don't know honestly. We don't have to merge all pages right now, but we should not start creating new (old) platform pages.